tests/integrations/server/raft_client.rs

```diff
@@ -67,25 +69,120 @@ fn test_batch_raft_fallback() {
     let mut raft_client = get_raft_client(&pool);
     let counter = Arc::new(AtomicUsize::new(0));
 
-    // Try to bind the mock server on a TCP port, and then do test.
-    for i in 0..100 {
-        let kv_service = MockKv(MockKvForRaft(Arc::clone(&counter)));
-        let port = 60000 + i;
-        let mut mock_server = match mock_kv_service(kv_service, "localhost", port) {
+    let service = MockKvForRaft(Arc::clone(&counter));
+    let (mock_server, port) = create_mock_server(service, 60000, 60100).unwrap();
+
+    let addr = format!("localhost:{}", port);
+    (0..100).for_each(|_| {
+        raft_client.send(1, &addr, RaftMessage::new()).unwrap();
+        thread::sleep(time::Duration::from_millis(10));
+        raft_client.flush();
+    });
+
+    assert!(counter.load(Ordering::SeqCst) > 0);
+    pool.shutdown().wait().unwrap();
+    drop(mock_server)
```

Comment:
Not similar with the context, but is this necessary?

Reply:
Otherwise we need to rename it to `_mock_server`.